website/source/docs/platform/k8s/injector/examples.html.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: "docs"
+page_title: "Vault Agent Sidecar Injector Examples"
+sidebar_current: "docs-platform-k8s-injector-examples"
+sidebar_title: "Examples"
+description: |-
+  This section documents examples of using the Vault Agent Injector.
+---
+
+# Vault Agent Injector Examples
+
+The following are different configuration examples to support a variety of
+deployment models.
+
+~> A common mistake is to set the annotation on the Deployment or other resource.
+  Ensure that the injector annotations are specified on the pod specification when 
+  using higher level constructs such as deployments, jobs or statefulsets.
+
+## Before Using the Vault Agent Injector
+
+Before applying Vault Agent injection annotations to pods, the following requirements
+should be satisfied:
+
+* Kubernetes auth method should be configured and enabled in Vault,
+* Pod should have a service account,
+* desired secrets exist within Vault,
+* the service account should be bound to a Vault role with a policy enabling access to desired secrets.
```

Comment:
Nice! I actually didn't know you could do this but it's in [the APA style guide](https://blog.apastyle.org/apastyle/2010/03/lists-part-5-bulleted-lists.html). Seems handy.